### PR TITLE
Enable various prow plugins for bazelbuild/rules_k8s

### PR DESCRIPTION
**Pull request by fejta**
/assign @BenTheElder @krzyzacy 
FYI @erain @mattmoor

### CODE REVIEW

File: prow/plugins.yaml [hunk 1]
```diff
@@ -160,6 +160,18 @@ config_updater:
...
+  bazelbuild/rules_k8s:
+  - approve  # Allow OWNERS to /approve
+  - assign  # Allow /assign and /cc
+  - blunderbuss  # Auto-assign people
+  - help  # Support /help and /good-first-issue
+  - hold  # Support /hold to delay merge
+  - lgtm  # Allow /lgtm
+  - lifecycle  # Allow /lifecycle stale
+  - size  # Auto-label size of PR
+  - trigger  # Allow people to configure CI jobs to /test
```
Comment (krzyzacy) [inline prow/plugins.yaml]: define trusted org at top, and do they have a prow job?

Comment (fejta) [reply prow/plugins.yaml]: Done.

No prow job yet, will work on that next

**Comment by krzyzacy** [comment]
is webhook configured?
/lgtm
/hold

**Comment by krzyzacy** [comment]
also you should enable cat/dog/yuks for them!

**Pull request merged**